### PR TITLE
core/webauthn: set webauthn interface in its app not in main.py

### PR DESCRIPTION
**Pull request by tsusanka**
I confirm this way the test passes on device.

### CODE REVIEW

File: core/src/main.py [hunk 1]
```diff
@@ -50,8 +50,7 @@ def _boot_default() -> None:
     if __debug__:
         import apps.debug
     if not utils.BITCOIN_ONLY:
-        if not __debug__ or utils.EMULATOR:
-            import apps.webauthn
+        import apps.webauthn
```
Comment (matejcik) [inline core/src/main.py]: why not move this up to the other `not BITCOIN_ONLY` block? do we need apps.debug first?

File: core/src/main.py [hunk 2]
```diff
@@ -71,8 +70,7 @@ def _boot_default() -> None:
     if __debug__:
         apps.debug.boot()
     if not utils.BITCOIN_ONLY:
-        if not __debug__ or utils.EMULATOR:
-            apps.webauthn.boot(usb.iface_webauthn)
+        apps.webauthn.boot()
```
Comment (matejcik) [inline core/src/main.py]: dtto

File: core/src/apps/webauthn/__init__.py [hunk 3]
```diff
@@ -18,4 +18,7 @@ def boot(iface: io.HID) -> None:
         __name__,
         "remove_resident_credential",
     )
-    loop.schedule(handle_reports(iface))
+    if not __debug__ or utils.EMULATOR:
+        import usb
```
Comment (matejcik) [inline core/src/apps/webauthn/__init__.py]: perhaps `import usb` can be on top level, seeing as `usb` is imported in all cases anyway

Comment (tsusanka) [reply core/src/apps/webauthn/__init__.py]: Can be..

File: core/src/main.py (hunk 1 above)
Comment (tsusanka) [reply core/src/main.py]: I'm unsure. I didn't touch the other code on purpose. Maybe it's just a leftover from the previous code: https://github.com/trezor/trezor-firmware/blame/07ef01f3e498449d7f32dc5df898897ed877d60b/core/src/main.py#L69

I'll try it and run tests.

**Pull request merged**